### PR TITLE
Update links to rocAL component

### PR DESCRIPTION
**Pull request by peterjunpark**
This PR
- Updates the link to rocAL on the `What is ROCm?` page
- Add a link to rocAL in API libraries

### CODE REVIEW

File: docs/reference/api-libraries.md [hunk 1]
```diff
@@ -23,6 +23,7 @@
 * {doc}`MIGraphX <amdmigraphx:index>`
 * {doc}`MIOpen <miopen:index>`
 * {doc}`MIVisionX <mivisionx:doxygen/html/index>`
+* {doc}`rocAL <rocal:index>`
 * {doc}`rocDecode <rocdecode:index>`
 * [ROCm Performance Primitives (RPP)](https://rocm.docs.amd.com/projects/rpp/en/latest/)
```
Comment (yhuiYH) [inline docs/reference/api-libraries.md:28]: Hmm... while you're here, are we able to make this relative link like the others?  or are there issues with that?

Comment (yhuiYH) [reply docs/reference/api-libraries.md:28]: It's done in what-is-rocm already

Comment (peterjunpark) [reply docs/reference/api-libraries.md]: Done. No issues.

**Review by yhuiYH — APPROVED** [review]
Thanks for the cleanup.  LGTM

**Comment by yhuiYH** [comment]
Please resolve and merge when ready

**Pull request merged**